docs_versioned_docs/version-ros2jazzy/ros/tutorials/simulator/install.mdx

```diff
@@ -16,15 +16,20 @@ It is highly recommended to install the simulator on a computer with a dedicated
 
 Follow the [Offboard Computer](../../installation/offboard_pc.mdx) setup guide.
 
-### Installing Gazebo Fortress
+### Installing Gazebo Harmonic
 
-Run the following commands in your terminal:
+:::note
+
+See Gazebo's [installation instructions](https://gazebosim.org/docs/harmonic/install_ubuntu/) for
+more information on installing and uninstalling Gazebo Harmonic.
+
+:::
+
+ROS provides a "vendor" package to install the default ROS/Gazebo pairing (e.g. ROS 2 Jazzy with
```

Comment:
Should we note that it is automatically installed with `clearpath_simulator`?

Reply:
Done.